libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -533,6 +538,11 @@ class YulUtilFunctions
 	/// signature: (array, newLen)
 	std::string resizeDynamicByteArrayFunction(ArrayType const& _type);
 
+	/// @returns the name of a function that cleans up elements of a storage byte array starting from pos
+	/// it will not copy elements in case of transformation to short byte array, and will not change array length
```

Comment:
Maybe clarify: It will not do anything for a short byte array. Also clarify: Does nothing if pos >= len (please also add that to the other function)

Reply:
Also clarify: Only clears starting from the first fully unused slot.

Reply:
... unless the new length is less than 32